### PR TITLE
Create Mainnet Citus in NA Cluster

### PR DESCRIPTION
**Pull request by jnels124**
create mainnet citus in na cluster

### CODE REVIEW

File: clusters/mainnet-na/mainnet-citus/sealed-secret.yaml
Comment (jnels124) [inline clusters/mainnet-na/mainnet-citus/sealed-secret.yaml:1]: Some double check this by creating the namespace in na and applying just this yaml please

Comment (steven-sheehy) [reply clusters/mainnet-na/mainnet-citus/sealed-secret.yaml:1]: Looks good

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
We also need to add mainnet-citus to `alert-slack.yaml`. Which it's apparently also missing from mainnet-eu

**Review by steven-sheehy — APPROVED** [review]

**Review by jascks — APPROVED** [review]
LGTM

**Pull request merged**